src/utility.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
```

Comment:
could probably throw all of these in the helpers file?  Or are we considering utilities different from helpers?